app/models/item.rb

```diff
@@ -1,5 +1,18 @@
 class Item < ActiveRecord::Base
   belongs_to :list
+  acts_as_list scope: 'list_id = #{list_id} AND done = false'
 
   validates :text, :list_id, presence: true
+
+  def mark
+    Item.transaction do
+      update_attributes done: !done
+
+      done? ? remove_from_list : add_to_list
+    end
+  end
+
+  def add_to_list
+    update_attributes position: list.items.where(done: false).count
+  end
```

Comment:
@vlasiak, That is named properly in respect of the symmetry (`remove_from_list`) :+1: But I'm wondering whether `move_to_bottom` [method](https://github.com/swanandp/acts_as_list#methods-that-change-position-and-reorder-list) would do the same as `add_to_list`?